app/routing/ArrivalsRouter.scala

```diff
@@ -60,7 +64,16 @@ class ArrivalsRouter @Inject() (
   def createArrivalNotification(): Action[Source[ByteString, _]] =
     route {
       case Some(VersionedRouting.VERSION_2_ACCEPT_HEADER_PATTERN()) => v2Arrivals.createMovement(MovementType.Arrival)
-      case _                                                        => v1Arrivals.createArrivalNotification()
+      case _ =>
+        if (config.enablePhase5) handlePhase5()
+        else v1Arrivals.createArrivalNotification()
+    }
+
+  private def handlePhase5() =
+    Action(streamFromMemory) {
+      request =>
+        request.body.runWith(Sink.ignore)
+        Gone(Json.obj("message" -> "Please use version 2 to create Arrival Notification"))
```

Comment:
It would be far better to create a `gone` `PresentationError` and use that here, in the same style as we do in phase 5.

We also need to have better messages than this, it's worth asking Fergal what good wording might be.